src/Subscribers/DoctrineEncryptSubscriber.php

```diff
@@ -175,19 +167,14 @@ public function postLoad(LifecycleEventArgs $args)
      */
     public function getSubscribedEvents()
```

Comment:
> That would break with the Doctrine interface so that's no possible.

Should be possible to [add hints to existing un-typed interfaces](https://3v4l.org/mHQjK):

```php
<?php

interface Foo
{
    function baz();
}

class Bar implements Foo
{
    function baz() : array { return []; }
}

var_dump((new Bar)->baz());
```